kernel/byterun/coq_interp.c

```diff
@@ -1529,6 +1571,20 @@ value coq_interprete
         }
       }
 
+      Instruct(CHECKASRINT63CONST1) {
```

Comment:
Where does this opcode come from?

Is it just because of `LSLINT63CONST1` and `LSRINT63CONST1` or is there some rationale to it? I don't know where these two are coming from either, so I intend to remove them in a subsequent pull request. So, no point in adding `ASTIN63CONST1` just o remove it later.

Reply:
> Is it just because of LSLINT63CONST1 and LSRINT63CONST1?

Yes. I was also confused about its use. I'll delete it then.